add redisClusterAsyncFormattedCommandToNode() function

andreasgerstmayr opened the pull request:
This PR adds a new `redisClusterAsyncFormattedCommandToNode()` function, which does the same as `redisClusterAsyncCommandToNode()` except it takes an already formatted command.

bjosv commented:
Thanks!
I think it would be nice if we now would call `redisClusterAsyncFormattedCommandToNode()` from `redisClusterAsyncCommandToNode()`.
This should reduce some duplicate code, WDYT?

andreasgerstmayr commented:
> I think it would be nice if we now would call `redisClusterAsyncFormattedCommandToNode()` from `redisClusterAsyncCommandToNode()`.
> This should reduce some duplicate code, WDYT?

+1, fully agree - I wasn't sure if I should update the existing code.
I've updated my commit now and removed the duplicated code.

zuiderkwast reviewed (commented):
LGTM.

@bjosv please have a second look.

bjosv commented:
Thanks!

andreasgerstmayr commented:
thanks for merging! :)

zuiderkwast commented:
I think you forgot to add yourself in the copyright comment in this file? ;-)

zuiderkwast commented:
... and possibly mentioning this function in docs? I guess the REAMDE is the only docs we have.

andreasgerstmayr commented:
Oops, forgot - addressed both issues in #40. Thanks for the notice!

zuiderkwast commented:
That's great @andreasgerstmayr. I guess we'll merge it after the weekend.
